sdk/search/Azure.Search.Documents/src/Batching/SearchIndexingBufferedSender{T}.cs

```diff
@@ -331,10 +333,20 @@ private async Task GetKeyFieldAccessorAsync(bool async, CancellationToken cancel
         /// </returns>
         internal async Task RaiseActionAddedAsync(
             IndexDocumentsAction<T> action,
-            CancellationToken cancellationToken) =>
-            await ActionAddedAsync
-                .RaiseAsync(action, cancellationToken)
-                .ConfigureAwait(false);
+            CancellationToken cancellationToken)
+        {
+            try
+            {
+                await ActionAddedAsync
+                    .RaiseAsync(action, cancellationToken)
+                    .ConfigureAwait(false);
+            }
+            catch
+            {
+                // TODO: Log any exceptions raised from async events that we
```

Comment:
Is there a tracking issue on this? I assume to add an EventSource?

Reply:
Fixed